packages/developer-portal/src/components/pages/settings/settings-organisation-tab/organisation-form/organisation-form.tsx

```diff
@@ -9,6 +10,11 @@ import { DeveloperModel, UpdateDeveloperModel } from '@reapit/foundations-ts-def
 import { selectSettingsPageDeveloperInformation, selectSettingsPageIsLoading } from '@/selector/settings'
 import { updateDeveloperData } from '@/actions/settings'
 
+const headingCss = css`
```

Comment:
This css is not needed - class names are in utils `flex justify-between`

Reply:
fixed as comment